dev_tools/packaging/verify-published-package.sh

```diff
@@ -83,6 +84,7 @@ for PYTHON_VERSION in python3; do
     if [ "${PYPI_REPO_NAME}" == "TEST" ]; then
         echo "Pre-installing dependencies since they don't all exist in TEST pypi"
         "${tmp_dir}/${PYTHON_VERSION}/bin/pip" install --quiet -r "${RUNTIME_DEPS_FILE}"
+        "${tmp_dir}/${PYTHON_VERSION}/bin/pip" install --quiet -r "${DEV_DEPS_FILE}"
```

Comment:
nit: can install both together
```suggestion
        "${tmp_dir}/${PYTHON_VERSION}/bin/pip" install --quiet -r "${RUNTIME_DEPS_FILE}" -r ${DEV_DEPS_FILE}
```
This might be more efficient, since all dependencies are resolved at the same time.

Reply:
done